### ISSUE TITLE
Make new image height crop optional

### ISSUE THREAD

**Issue by fenarinarsa** [bluesky-social/social-app#5263]
**Is your feature request related to a problem? Please describe.**

With PR #5129, v.0.91 changed the way medias are displayed, making some users and artist frustrated.

As a matter of fact:
1/ I also loved having big vertical images that were not cropped and took the full column width. Now nearly all medias got cropped *and* reduced in height and width for almost no reason, which makes Bluesky less appealing for artists and "gallery" feeds.

2/ I didn't understand the meaning of the new icon popping on nearly all medias until I checked the PR. It would be interesting to check how many users get it.

3/ Now I'm urged to open every media that has the crop icon to check them, which breaks the feed scrolling. Also, most of times it's only a few pixels that have been cropped out, because artists usually don't make 3:4 illustrations. I think most people won't get that the picture has been cropped, though (because of 2).

**Describe the solution you'd like**

Revert to the old display and make the new crop optional.

**Describe alternatives you've considered**

Allow aspect ratios up to 9:16 (standard) before cropping. Artists usually make art taller than 3:4 but not taller than a smartphone standard aspect ratio.

Make the old display optional. Most people will have the crop icon popping though and it breaks the nice "art gallery" feeling on Bluesky, especially "picture only" feeds.

**Additional context**

PR: https://github.com/bluesky-social/social-app/pull/5129
https://main.bsky.dev/profile/kimhu.bsky.social/post/3l3ublhsqsx2w

This one like many others gets cropped for only a few lines:
https://main.bsky.dev/profile/datcravat.bsky.social/post/3l3nmldyixi2n

Keke's baguette post:
![image](https://github.com/user-attachments/assets/95b53cf8-cc1e-4356-b111-fca5a61f9b58)

**Comment by kinokonoonnanoko** [comment]
I concur. Scrolling through the feeds and being able to see images uncropped without an extra click to expand them was one of the best features of Bluesky. The new ratio messes with this experience, and worst case renders some posts (especially memes with text on top/bottom) unreadable on first glance. If a crop is needed, it should only crop very long images over 9:16 ratio as suggested and/or be an optional function to be turned on in the settings.

**Comment by Bossett** [comment]
I think related to #4336 that I raised re: avoiding images that dominate the timeline - I think it's maybe overcorrected a little now (I think a little crop to fit a ratio limit would be acceptable?) in the new version.

Ideally (for me) something that caps the size, and just scales down to a max? That may affect the overall balance of the app though (i.e. portrait or skinnier images would have a lot of space either side).

**Comment by Tywele** [comment]
I agree with this suggestion. Also cropping images leads to more clicking on the users side.

**Comment by csuhta** [comment]
+1 to kinokonoonnanoko's suggestion, the image should only be cropped by the app view if they are extremely tall

**Comment by BetoRJR** [comment]
Completely agree with the ask, since quite a few artists seem to be having difficulty adjusting to this new ratio crop. If we can’t have a toggle, perhaps adjusting the proportions might help.

**Comment by Sininini** [comment]
Absolutely agree. I don’t like to have to open so many images just to see them fully and sometimes you can even miss important context due to crop.

Also it’s quite ridiculous to crop just a few pixels because image was barely over the limit.

**Comment by neomints** [comment]
While I understand the need for some kind of limit to avoid issues that sites like tumblr had with images that would span multiple post-lengths long, where people would purposefully add way too many of them to annoy other users, I feel like it would make the most sense to at the least allow phone-aspect ratio images before being cropped like was previously given as an example, since that's really unlikely to ruin the scrolling experience

**Comment by grembowski** [comment]
16:9 is fairly standard these days for aspect ratios. Portrait oriented pictures in that ratio shouldn't be cropped.

**Comment by SpriterGors** [comment]
The crop feature was just implemented on my side and I don't get it, most of the times the crop isn't even helping because most people don't make extremely tall pictures either, as described.

+1 for @kinokonoonnanoko for me too, to me their suggestion makes the most sense.

**Comment by Bossett** [comment]
I do think 9:16 can end up way too tall if the width is scaled to the viewport (e.g. a quote of an image post is more than a phone screen tall for a single post) - but some kind of max height constraint could sort that out - scaled to within a specific bounding box. That seems to be the approach taken by threads - that mostly just scales the image (though applies a crop sometimes on weird ratios).

**Comment by Nemo2342** [comment]
I agree as well; one of the (many) things that made Bluesky better than that other site was being able to see the work artists post without it being terribly cropped.

**Comment by RoboLeader** [comment]
Agree, this needs to be reverted or at least made optional.

**Comment by fenarinarsa** [comment]
> I do think 9:16 can end up way too tall if the width is scaled to the viewport (e.g. a quote of an image post is more than a phone screen tall for a single post) - but some kind of max height constraint could sort that out - scaled to within a specific bounding box. That seems to be the approach taken by threads - that mostly just scales the image (though applies a crop sometimes on weird ratios).

I don't like Threads when it comes to media display, I actually think it's the worst you could do. I guess it's a matter of taste :)

I prefer the media to take the max width instead of being weirdly left aligned with a blank part on the right, even if it ends up bigger than the viewport in my feed. I think that's what happened on Bluesky until the last update, and it never bothered me. It's very different from what X and Threads offer, and closer to what Mastodon and Misskey do. Both have settings related to media cropping btw.

**Comment by BagelHero** [comment]
I also prefer that if things are to be cropped it covers the maximum timeline viewport size... I think, personally, a good solution to cropping being unfriendly to artists is to allow manual crop area selection, like old school forum icon uploads. It allows for some sense of intention and framing to remain and prevents cropping out important areas.

**Comment by FaustKatt** [comment]
Adding an auto-crop annoys me on several levels. As an artist I've observed it making minor, frankly pointless crops to my images that are in very normal formats. I have no control over this which means that composition on bluesky is going to be rubbish unless I adhere to its specific peculiarities. If it has to happen at all it ought to at least be controlled by the uploader.

As a user reading the site it burdens me with having to frequently stop and expand an image like a meme or other similar thing that I had **zero need to see up close**, because now I know something is missing from it but have no idea what. Mostly it's minor, pointless crops. Other times it's something like a small screenshot which is now missing context until I expand it, but which would have been perfectly legible under the old system.

This is a pointless time waster during normal use of the site. If I could turn it off I'd do it once and never think about it again, just like I never missed this "feature" in the first place.

**Comment by vertigris** [comment]
I agree with this. It was really nice to have the images full width and uncropped. The feed looked a lot more balanced and less chaotic than the like of X and Threads. 
Of course there needs to be a limit, but min height should at least be 9:16. If the thought behind this was to save scrolling time, having QRTs be full sized is more of a problem (even though I still like it that way personally).

**Comment by JayIsPainting** [comment]
As an artist, the fixed viewport degrades my ability to communicate my work. As a user, it degrades the usability of the site by  adding friction and extra clicks. This change should be rolled back or at minimum, made an option in setting like auto play gif & video.

**Comment by yevvie** [comment]
modern design dictates **(uniform) scale don't crop**. this is a blow to artists' marketing, self-promotion and overall display of the artworks / photos that are designed with specific viewing experience in mind that is being hindered by this design choice. now 

you cannot easily read webcomics with this. you also cannot capture anyone's attention and present your work in good sense, as even basic grid rules stop applying and attention is being pushed into different, less refined parts.

on the side note, first reply changes, now this, the platform is appearing more and more anti-artists and i don't think this is a good direction to go with.

this is like, experience-breaking change for many artists and art enjoyers.

**Comment by gaearon** [comment]
Thanks for feedback. Can yall drop a few links to specific profiles and/or posts where the new behavior is noticeably worse? It would also help to clarify whether it’s the web behavior or mobile app that’s frustrating (since they use a slightly different algorithm).

**Comment by fenarinarsa** [comment]
Example (web version)
bad crop + scaling + left alignment

I'd say it's equally frustrating than on X.

And I don't think it's a good idea to compare X and Bluesky. IMO it's better for Bluesky to have its own "personality". Scaling and cropping images to constraint the height of all posts kind of killed the "wow" effect it had before. It's important because artists are amongst the hardest kind of people to persuade to make a switch since they have not a lot of time on their hands but have a lot to lose by changing their social networks. The artists I personally know don't like crops because it forces them to export a different version of their pics for every social network they're on.

![image](https://github.com/user-attachments/assets/d72cb3c6-ed58-4e35-8b58-6f7daf2c063b)

**Comment by BagelHero** [comment]
**Examples**
![image](https://github.com/user-attachments/assets/d774e30e-f500-47fd-99cb-ecbb3051ad80)
Dislike that this is shrunken and shifted off to the left-- the intention perhaps to limit impact of cropping tall images... instead it just hurts the intentional framing AND makes viewing it full size sort of not worth the time. Shifting it off to the side is also unappealing _(and on making the window smaller, possibly a bug because it's fine if the window is small enough to make both sidebars disappear)_. would rather it cut off MORE if it's going to crop.

Then...
![image](https://github.com/user-attachments/assets/46c20a4c-92f2-4dee-80b6-fd5dfe608263) 
vs the full size image...
![image](https://github.com/user-attachments/assets/ca66483e-e401-4b47-a085-e75b28d5796e)
good example of the crop both hurting the intricate frame of this work as well as just... making it feel kind of silly to have cropped in the first place. it almost feels like spot the difference at times but stand out in this example because it has no bleed.

All of the most obvious examples I've spotted so far have been works for works for print, photos or comics, like above. Most, at least, are easy to predict and design for. I fully believe works made for A5/A4/A3 dimensions should never crop, I believe currently B5/B4/B3 are probably just stubby enough to not get cropped, but they also shouldn't. As pointed out above, the common 3:4 portrait gets cropped, which is a bit wild. 16:9 is probably too long to get away with... but that comic book cover is really suffering, I feel like whatever proportion that cover is -- that's the sort of upper limit for length for me before I start thinking crop is needed. I understand thats crazy subjective though.

> Also, just to clarify — is the new behavior worse/different than on twitter, or are you equally frustrated with the behavior on twitter? Thanks.

In agreement with others that Twitter sucked for this also, and their regular adjustments to optimal posting size made it feel like we were regularly aiming at moving goalposts when trying to make infographics/hiring/available for work posts-- those are kind of boring in the first place so if its hard to grasp the info at a glance people tend not to click through at all. I think limitations are okay, they just need to either be somewhat controllable **or** reliably predictable.

**Comment by Alaspooralice** [comment]
I agree I don't want the cropping of images done automatically, I want to see the whole image so I don't have to click everything.

**Comment by ProfBV** [comment]
I am supporting the request to not crop image height on the timeline, as an artist

**Comment by str4d** [comment]
As I understand it, the purpose of cropping is generally so the average user's main / mixed-source feeds is not overwhelmed by a few posts (e.g. #4336). For me it's also important from a _health_ perspective: the longer individual posts in my feed are, the more often I have to use my thumb to scroll on my phone, which can aggravate RSI.

When scrolling on my phone I'll generally see three or four different text posts (sometimes five!) on the screen at a time. But even with the current cropping rules in version 1.91, the moment one of the posts contains a _portrait_ image I'll see 1.5 posts max on the screen. And if the post is a quote post with an image of a post that also has an image, then I won't even see the entire quote post at once on the screen.

So as a browsing user, I _do_ want automatic cropping of images and a more compact view for quote posts - in my main feed. But if I'm in a view where I _expect_ to primarily see image content, then I don't mind the additional scrolling (and may intentionally be scrolling less because I want to stop and look at the images). The main example here is the Media tab for a user, but the Posts tab for an artist (who might be regularly reposting art from other users in addition to their own) would also be the case, as would an art-specific feed, or an alt account specifically for following artists.

So my question here is, how viable would it be from a UX perspective to have different default crop behaviours in different views? The main downsides I imagine here is _predictability_ on the part of both the user uploading an image and the user browsing a view, but if e.g. there are just two defaults ("compact" and "larger") then that might be manageable. A config option like has been suggested upthread could also allow users to toggle between the two defaults in their main feed.

**Comment by Irajiack** [comment]
> For me it's also important from a _health_ perspective: the longer individual posts in my feed are, the more often I have to use my thumb to scroll on my phone, which can aggravate RSI.

Maybe an option to be displayed cropped or not should be on the _user_ side and not server or poster side. If some people prefer seeing everything, they'll be annoyed if the auto crop is installed. But for people that prefer when stuff are cropped, seeing everything will be annoyed, so either way you're kinda making mad a part of the user base, unless you offer them the options to see the images as cropped or not? 

Because i do get that some people does find annoying (or even painful in certain cases) to scroll a lot just to go past _one_ post... Which is in part what a micro blogging platform is for, seeing the most posts in the view port size of the screen. 

Not sure how much harder that would be to implement such kind of option, but it'll defo be more work than just choosing one or the other.

**Comment by gaearon** [comment]
I’d like to reiterate that it’s really important we nail the defaults — adding options doesn’t do anything for the majority of people (either on consuming or producing side) and there would still be a debate about the default. So let’s focus the discussion and examples on the default, not on the toggles.

**Comment by doodlemancy** [comment]
Thank you for listening to feedback on this and taking it seriously! I'd like to chime in and say that letting the cropping be toggled off on the user side is something I'd like to see, but I also don't feel like that would make a good solution on its own-- because then that's something we have to _ask_ our audiences to do in order to properly view our work. Then you end up with some people who have the crop on, some people who have it off, and you end up having to do the same work anyway because you have to account for the people who are going to see the terrible crop of your images if they haven't changed their settings.

Another BIIIIIIIIGGGGGG problem with this kind of cropping is that you end up cropping out people's signatures partially or fully-- and then people who don't know how to save images on their phones screenshot our art, it loses the signature, maybe they post it somewhere else... it's a whole thing.

Also... I've been on bsky a year now and I have never once had a problem with a too-tall or too-wide image. Rounding the corners on images isn't making the website look better, it's mostly just making everything look a little awkward and out of place, and driving artists bonkers.

![bad crop](https://github.com/user-attachments/assets/3ef28757-528b-4376-be39-2cf20800221f)
![no crop](https://github.com/user-attachments/assets/ba039948-a9fc-4837-a7d7-41b6bb3f907d)

This is on desktop. The goal here is... unclear to me at best. This is a tall image with admittedly weird proportions, but it's not so tall or weird that it was impossible to scroll past comfortably, and now it just looks messy. It's not the worst that's ever been done to my art by a website (twitter would have sliced it down to only show her from like, chin to hips or something back when things were at their worst there) but it's bothersome. Look at my poor signature! And it just took off a fraction of the top of her head.

My app hasn't updated yet, and it looks absolutely fine there:

![Screenshot_20240911-100341](https://github.com/user-attachments/assets/b44d4bfd-ee3a-4817-97f6-31800e5a5c8e)

There's just... no need for this crop to be happening on the vast majority of images, sorry. Since this update rolled out, I haven't seen a single cropped image that actually needed to be cropped. Just a lot of awkward edges on stuff. :(

The best solution, in my opinion, is to figure out where you want to start containing the outliers-- and then, most importantly-- let people choose _how_ their actually-too-tall images are cropped. But leave everyone else's single-image posts alone! Pretty please! Full-width is cool and good! Thank you! (I hope this didn't sound too grouchy lol)

**Comment by RossBrunton** [comment]
For more feedback, the Mastodon fork I use has a setting for "Letterbox" media which looks like this:

![image](https://github.com/user-attachments/assets/1e83eb3b-23c2-49ea-8667-61470d43fff2)

This could be a reasonable compromise and sensible default for allowing the full image to be visible whilst also not allowing images to stretch out feeds. At least, more sensible a default than cropping to focus on a woman's breasts and crotch at any rate.

Of course, long term, I think the best is to have this configurable for the user, and allow uploaders to manage their own cropping.

**Comment by yevvie** [comment]
@gaearon 
just as doodlemancy mentioned in  , signatures / watermarks are getting nuked with cropping too

![Screenshot 2024-09-11 215558](https://github.com/user-attachments/assets/2665db7b-7b77-4277-aeaf-1d965ec50f38)
![Screenshot 2024-09-11 215833](https://github.com/user-attachments/assets/2a5babf8-9b60-4043-b495-2018b77f8efb)
![Screenshot 2024-09-11 215553](https://github.com/user-attachments/assets/b2e34f48-9e79-45ce-9764-27b7b7ad8e2c)

**Comment by xenianth** [comment]
I agree that the crop is too extreme, formats strangely by not filling post field horizontally, and is detrimental to a smooth experience for users. It unnecessarily dings artists whose work and income is often dependent on their social media presence. Echoing that the cut-off should be 9:16.

Examples, mostly of my own posts, but also one recent other one that clearly shows how it negatively affects both user and poster experience.

![02](https://github.com/user-attachments/assets/84cad01e-b546-4401-bad6-f03dad1d1956)
![01](https://github.com/user-attachments/assets/0eb54ad6-f94b-44d4-87e9-106c64d8141d)
![03](https://github.com/user-attachments/assets/208bb18a-fe2d-40a1-9ea0-b0cf84741715)
![04](https://github.com/user-attachments/assets/9536cc0d-7c3c-4ff1-aba4-650293c364aa)

**Comment by gaearon** [comment]
We'll give 16:9 cutoff a try and see how that feels against examples in this thread, thanks everyone again!

**Comment by HeyItsLollie** [comment]
I'm a big fan of the option that RossBrunton shared just a few posts up — scale the image to fit within an embedded viewport, and center the whole image within the post.

One thing Twitter does that I _do_ like, is filling the black space with a scaled & blurred copy of the image. A quick mock-up example:
![image](https://github.com/user-attachments/assets/260cd9f5-1706-4121-945d-278b38c4ecd9)

In addition, I would suggest giving users the ability to set our own preview crops. Let us choose whether the image is scaled to "Fit" or "Fill" the preview viewport. For users who select "Fill", provide an option to override the automatic crop so that we can frame the image manually.

And then finally: Give users an additional setting in Following Feed Preferences, to toggle whether images have smaller previews, or are allowed to be big and expand the post, as a way to side-step these crops/previews.

**Comment by EkanaStone** [comment]
I'm not sure if this is the issue to say this but I also think videos are being cropped very egregiously on mobile.

**Comment by arsatiki** [comment]
https://bsky.app/profile/comiccontext.bsky.social/post/3l3wh2xkt2f2l

Comics Without Context regularly posts tall images, with the punchline now getting cut off. I’d like to see images uncropped at least to 2:1 aspect ratio.

**Comment by captainharrie** [comment]
> While I understand the need for some kind of limit to avoid issues that sites like tumblr had with images that would span multiple post-lengths long, where people would purposefully add way too many of them to annoy other users, I feel like it would make the most sense to at the least allow phone-aspect ratio images before being cropped like was previously given as an example, since that's really unlikely to ruin the scrolling experience

I would like to add that I think tumblr actually has the best solution for extremely tall posts, specifically added to combat that behaviour.

![Screenshot_20240912-102245](https://github.com/user-attachments/assets/6c449a27-c9f5-4185-866f-ce391993e0f2)

You can enable the option to shorten long posts, which rather than popping out an image into a lightbox that you then also have to click out of which is frustrating and stops you in your tracks whole scrolling, just shortens the height of the post with a button to expand it. This allows you to easily scroll past long posts that do not catch your attention, and also lets you view long posts inline in your timeline with a single click that doesn't interrupt your scrolling. It's just overall better UX than cropping the image and expanding it in the lightbox.

Before this was a native tumblr experience, i used an extension for the same feature. For me, 150vh was the ideal point for cutting long posts, but tumblr is a longform blogging platform so posts are expected to be longer. I'd maybe say 100vh on bluesky..?

---

Anyway if we must have cropped images, then I do think at the very least we should be able to choose the focal point of the crop like on mastodon.

**Comment by estrattonbailey** [comment]
Hey y'all, thanks for all this feedback, we hear you! We want to make Bluesky a great place for all types of content, without negatively affecting the experience of users as a whole.

To that end, we've re-tooled the image cropping and aligned on the following rules, which should handle the vast majority of cases pretty well.

**Sizing rules:**
- desktop web
  - height is limited by a 1:1 square container (see red dashed container in screenshot)
  - images taller than 1:2 will be cropped (but 9:16, 2:3, 3:4, etc will not)
- mobile
  - height is limited by a 9:16 container (see red dashed container in screenshot)
  - images taller than 1:2 will be cropped (but 9:16, 2:3, 3:4, etc will not)

![image sizing](https://github.com/user-attachments/assets/88bdcbf3-fde2-4917-bfc2-3c5409d158b1)

## Examples
![CleanShot 2024-09-12 at 10 00 34@2x](https://github.com/user-attachments/assets/c242e8d4-fd9c-4a40-9344-05d391ab686c)
![CleanShot 2024-09-12 at 09 59 54@2x](https://github.com/user-attachments/assets/19cd0e25-3d56-4ccd-87f6-6db0e56f545c)
![CleanShot 2024-09-12 at 10 00 38@2x](https://github.com/user-attachments/assets/e55479f3-6900-49b2-ae87-85
b83146f3fe)
![CleanShot 2024-09-12 at 10 00 52@2x](https://github.com/user-attachments/assets/21158822-9e12-4305-95f1-661738754567)
![CleanShot 2024-09-12 at 10 01 11@2x](https://github.com/user-attachments/assets/24df47e9-049e-42cd-a979-f379e9fa6b96)
![CleanShot 2024-09-12 at 10 01 30@2x](https://github.com/user-attachments/assets/f27ddee4-6cf4-44e6-ae69-29fff3c4e42c)
![CleanShot 2024-09-12 at 10 01 50@2x](https://github.com/user-attachments/assets/89df6646-72e5-46ac-b1a3-4bc91857e844)
![CleanShot 2024-09-12 at 10 02 01@2x](https://github.com/user-attachments/assets/cdd6af84-2bbd-4b3d-b035-f45c5838b130)
![CleanShot 2024-09-12 at 10 02 18@2x](https://github.com/user-attachments/assets/09c49036-4676-44e2-8f26-c28a5d792dea)
![CleanShot 2024-09-12 at 10 02 28@2x](https://github.com/user-attachments/assets/9d6d972d-2918-468a-9114-d209e45132d8)
![CleanShot 2024-09-12 at 12 24 50@2x](https://github.com/user-attachments/assets/b63705e7-2e21-4d77-9210-ce67d754cc90)
![CleanShot 2024-09-12 at 12 27 38@2x](https://github.com/user-attachments/assets/3d890509-2953-4590-945e-a4650c96aa74)

**Comment by jh4c** [comment]
There's a huge issue with this new crop when it comes to displaying linked content, the picture pulled from the link is never going to have been optimised to fit BlueSky's layout so you end up with this:
<img width="612" alt="Screenshot 2024-09-12 at 04 13 32" src="https://github.com/user-attachments/assets/0cd88d17-f63b-4ba4-9c63-d16f0176a7b9">
Not a very flattering look for the person pictured!

**Comment by BagelHero** [comment]
> Hey y'all, thanks for all this feedback, we hear you! We want to make Bluesky a great place for all types of content, without negatively affecting the experience of users as a whole.
> 
> To that end, we've re-tooled the image cropping and aligned on the following rules, which should handle the vast majority of cases pretty well.
>-snip-

This sounds and looks promising. Still not a huge fan of the left alignment on desktop, but seeing it in context of other posts I'm sort of understanding the decision, and could possibly just acclimatise. Happy to give it a chance given the response time and predictability! Thank y'all 😊

**Comment by SpriterGors** [comment]
The updates seem to be interesting, thanks for the update. Would it be possible for the site to notify us if a crop happens, before the post is made? Something like "Image too tall; image may be cropped" so that we can adjust the picture.

**Comment by fenarinarsa** [comment]
> To that end, we've re-tooled the image cropping and aligned on the following rules, which should handle the vast majority of cases pretty well.

Thank you for the feedback.

Naive question, are the rules based on the viewport's size or is it an app/web fixed rule?
I know it's not the way most people use Bluesky, but I use it on vertical monitor and also use the web version on iPad :)

Else I guess I'd need to wait for a "Better Bluesky" extension 😌

**Comment by ameknite** [comment]
@estrattonbailey 
The new sizing rules are great. 
But they don't work well in tablets.
Tablets are generally 4:3, 3:2 or 16:10

Examples:
Apple iPad Pro 11 (2024) - 1668 x 2420 pixels, around 3:2 ratio
Samsung Galaxy Tab S9 - 1600 x 2560 pixels, 16:10 ratio

In a 16:10 tablet:
![1-2](https://github.com/user-attachments/assets/4287a459-3df4-4021-a66d-4b2d0548541f)
![9-16](https://github.com/user-attachments/assets/853176ae-fb11-4fe8-8257-fc8c2c0ca7d0)
![3-4](https://github.com/user-attachments/assets/74916810-f50b-4275-be2b-fe9f07b3dd13)

**Comment by SpriterGors** [comment]
I don't know if link pictures are subject to the same cropping rules but here's a case of an unfortunate crop:

![image](https://github.com/user-attachments/assets/81720f25-cd2d-4626-ac26-b8485c0e341f)

**Comment by mschwendener** [comment]
<img width="601" height="483" alt="Image" src="https://github.com/user-attachments/assets/e1ff0aa5-61be-4725-a237-7803ef2b3b99" />

More examples for the problem in the link card here https://github.com/bluesky-social/social-app/issues/1118 and here https://github.com/bluesky-social/social-app/issues/7443

**Comment by mschwendener** [comment]
<img width="601" height="471" alt="Image" src="https://github.com/user-attachments/assets/9d75e99a-02e6-49ca-8fe2-14a181b57827" />

**Comment by mschwendener** [comment]
https://bsky.app/profile/thethinkery.bsky.social/post/3mccmfssox22b 

<img width="661" height="622" alt="Image" src="https://github.com/user-attachments/assets/f57f4edd-a95d-41f2-ad5e-871220106749" />

**Comment by mschwendener** [comment]
https://bsky.app/profile/fotofacade.bsky.social/post/3mmthisaqla2a 

cc @abenzer – related to https://bsky.app/profile/alexbenzer.com/post/3mlvrfztuvu25 "why is post editing important to you?"
 
<img width="603" height="505" alt="Image" src="https://github.com/user-attachments/assets/46b0ac08-a183-48b7-9d70-f8f53ec14662" />

**Comment by mschwendener** [comment]
https://bsky.app/profile/london.gov.uk/post/3moiu6pge2s24 

<img width="597" height="484" alt="Image" src="https://github.com/user-attachments/assets/d616843f-bf32-4c86-8fe1-05c6da7e3647" />

**Comment by SpriterGors** [comment]
<img width="600" height="821" alt="Image" src="https://github.com/user-attachments/assets/73ee1bfd-1220-4f22-a4da-a84ea5b23a41" />
<img width="600" height="135" alt="Image" src="https://github.com/user-attachments/assets/01d4641b-dcc9-4d5e-9a8d-d5f4b3ec4d87" />

Current crop method causing friction to creators.

**Comment by mschwendener** [comment]
https://bsky.app/profile/scalzi.com/post/3monssanmcc2g

<img width="594" height="544" alt="Image" src="https://github.com/user-attachments/assets/8706c62e-22c0-4785-a2ca-0c05558dd43b" />

**Comment by mschwendener** [comment]
https://bsky.app/profile/parkerlibcccc.bsky.social/post/3mpybpucev22g 

<img width="657" height="677" alt="Image" src="https://github.com/user-attachments/assets/d8f8db5f-7837-4fa7-888c-44f350ab70d0" />

**Comment by mschwendener** [comment]
@mozzius 

Closed as completed because somebody worked on it and a fix will come with the next update?

**Comment by mozzius** [comment]
> @mozzius 
> 
> Closed as completed because somebody worked on it and a fix will come with the next update? 
>  

The external card opengraph crop problem is a separate issue. I believe there's already an open issue for it

**Comment by mschwendener** [comment]
Thank you, @mozzius  

Can you please link to the other ticket here, and check that it's open?

**Comment by mschwendener** [comment]
If you mean this https://github.com/bluesky-social/social-app/issues/1118 from August 2023 please reopen it and assign it to somebody. Thank you.

**Comment by mozzius** [comment]
https://github.com/bluesky-social/social-app/issues/5012

**Comment by mschwendener** [comment]
Thank you.
